mask/eolearn/mask/cloud_mask.py

```diff
@@ -371,8 +85,8 @@ def __init__(self,
                  mask_feature='CLM_INTERSSIM',
                  mono_threshold=0.4,
                  multi_threshold=0.5,
-                 average_over=1,
-                 dilation_size=1):
+                 average_over=4,
```

Comment:
I know that `get_s2_pixel_cloud_detector` function had these default but `s2cloudless.S2PixelCloudDetector` has defaults `average_over=1, dilation_size=1` ([link](https://github.com/sentinel-hub/sentinel2-cloud-detector/blob/master/s2cloudless/cloud_detector.py#L50)).

I really don't know what is better here and I don't remember what was the reason for choosing certain parameters.

Reply:
Neither do I. 

I chose this values because that meant the least changes in the existing example notebooks. We have also often seen people using s2cloudless with other values (not these), and they then claimed s2cloudless to be worse than it is. 

As for the reasons: I'd say they were selected empirically.

Reply:
Ok, that makes sense. We can leave it as it is now.